Assets/Forg.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Forg : MonoBehaviour
```

Comment:
Could you please add comment what this class does

Reply:
Done